### PR TITLE
Add build-cesium task, and use dependency tree

### PR DESCRIPTION
**Pull request by ggetz**
Add back `build-cesium` task, and add add module for determining the dependency tree of files specified.

Part of https://github.com/AnalyticalGraphicsInc/gltf-pipeline/issues/330

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,26 @@ To run ESLint automatically when a file is saved, run the following and leave it
...
+
+Some functionality of gltf-pipeline is used by Cesium as a third party library. The necessary files can be generated using:
+
+```
+npm run build-cesium
+```
+
+This will output a portion of the gltf-pipeline code into the `dist/cesium` folder, reformatted into AMD style for use with RequireJS and Cesium in the browser.
+
+### Building for other integration
```
Comment (lilleyse) [inline README.md]: Fine to remove this section since the `build-cesium-combine` task isn't used anymore.

**Comment by ggetz** [comment]
@lilleyse Thanks, updated!

File: lib/findAccessorMinMax.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ var ComponentDatatype = Cesium.ComponentDatatype;
 var defined = Cesium.defined;
 
 var getAccessorByteStride = require('./getAccessorByteStride');
-var getComponentReader = require('./getComponentReader.js');
+var getComponentReader = require('./getComponentReader');
```
Comment (lilleyse) [inline lib/findAccessorMinMax.js]: Ah, thanks for noticing this. I saw a couple other cases of this and pushed the changes.

**Comment by lilleyse** [comment]
Thanks @ggetz, looks good.

**Pull request merged**